### ISSUE TITLE
Sentence is a bit hard to follow. Suggest rewording as shown: 

### ISSUE THREAD

**Issue by dig-comments** [usnistgov/800-63-3#859]
**Organization Name (N/A, if individual)**: National Security Agency

**Organization Type**: 1
 
**Document (63-3, 63A, 63B, or 63C)**: 800-63B

**Reference (Include section and paragraph number)**: 6.1, 1
 
**Comment (Include rationale for comment)**:  Sentence is a bit hard to follow. Suggest rewording as shown:
 

"Authenticators MAY be issued (provided) by CSPs as part of the enrollment or other processes. In other cases, the subscribers MAY provide their own authenticators, such as software or hardware cryptographic modules. For these reasons, this guideline refers to the binding of an authenticator rather than the issuance of an authenticator. This does not exclude the possibility that an authenticator could be issued at the time of binding as well."

Original text: "Authenticators MAY be issued (provided) by a CSP as part of a process such as enrollment; in other cases, the subscriber MAY provide their own, such as software or hardware cryptographic modules. For this reason, this guideline refers to the binding of an authenticator rather than the issuance, but this does not exclude the possibility that an authenticator is issued as well."

**Suggested Change**: Make suggested change.
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by jimfenton** [comment]
Agreed that the wording can be cleaned up; made some additional adjustments to suggested wording.
